css/index.css

```diff
@@ -1 +1,413 @@
+		font-size: 2.5rem;
+		font-weight: bold;
+}
+
+ /*       widgets    */
+
+.services .widgets{
+    padding:0;
+	height: auto;
+	display: flex;
+	flex-wrap: wrap;
+	margin-top: 35px;
+
+	justify-content: space-between;
+	align-items:  space-between;
+	align-content: space-between;
+	border-bottom: 1px solid black;
+}
+.services .widget-box{
+    width: 45%;
+    margin-right: 15px;
+	height: auto;
+	background: #F4F3F4;
+	border: 1px solid black;
+	margin-bottom: 40px;
+	display: flex;
+	flex-direction: column;
+	justify-content: flex-start;
+}
+.services .widget-box h2,.services .widget-box p{
+	margin: 25px 10px 0 20px;
+}
+.services .widgets .widget-box .button{
+	background: white;
+	border: 1px solid gray;
+	text-align: center;
+	width: 120px;
+	height: 35px;
+	padding: 10px;
+	font-size: 1.2rem;
+    border-radius: 5px;
+    margin-bottom: 20px;
+}
+
+
+/*     foooter    */
+.services .footer-info{
+	display: flex;
+	margin-top: 40px;
+}
+.services .footer-info section{
+	margin-right: 50px;
+
+}
+.services .footer-info li{
+    font-size: 1.6rem;
+	line-height: 2em;
+	list-style-type: square;
+  list-style-position: inside;
+}
+
+
+/*        responsive      tablet      */
+
+@media (max-width:800px) {
+  
+
+    header{
+        flex-direction: column;
+        justify-content: center;
+        align-items: center;
+        
+
+    }
+    header nav{
+        order:1;
+        width: 100%
+    
+    }
+    
+    header nav a {
+      width: 100%;
+      
+    
+      
+      display: block;
+      text-align: center;
+    }
+    .cta{
+        justify-content: center;
+        margin-top: 0;
+    }
+    .cta img{
+        display: none;
+    }
+    .cta .cta-text{
+    align-self: center;
+    margin-top: 80px;
+    }
+    main, .contact{
+        margin-left: 10px;
+        margin-right:10px;
+    }
+    .middle-img{
+        width: 100%
+        
+        
+    }
+
+    /*  services-responsive */
+
+    .services{
+        width: 96%;
+        margin: 2%;
+        
+    }
+    .services img{
+        width: 100%;
+        height: 0 auto;
+    }
+    .services .footer-info{
+        display: block;
+    }
+    .services .footer-info section{
+        margin-bottom: 30px;
+    }
+  
+  }
+  
+  /*    mobile  */
+  @media (max-width:500px) {
+    header nav {
+        flex-direction: column;
+        margin: 20px;
+        align-items: center;
+        border-top: 1px solid gray;
+      }
+      header nav a {
+        font-size: 1.8rem;
+        width: 100%;
+        padding: 20px;
+        border-bottom: 1px solid gray;
+        display: block;
+        text-align: center;
+      }
+      .home .cta .cta-text{
+        margin-top: 30px;
+      }
+      
+      main .top-content {
+        display: block;
+        margin-top: 50px;
+      }
+      main .top-content .text-content{
+          margin-bottom: 50px;
+          width: 100%;
+      }
+      main .bottom-content {
+        display: block;
+        /* margin: 20px 0 0 0; */
```

Comment:
Consider removing code that you're not using; also keep in mind that you can shorten _TRBL_ into _T LR B_ here: `margin: 20px 0 0;`